tests/test_training.py

```diff
@@ -96,3 +109,43 @@ def test_ipu_cli_training(self, load_type):
 
                 assert _IPU_AVAILABLE is True
                 self.call_cli_with_overrides("ipu", "16-true", load_type)
+
```

Comment:
```suggestion
@pytest.mark.ipu
```
Need to make sure this is skipped for the non-ipu tests